content/docs/transforms/quickstart.md

```diff
@@ -0,0 +1,124 @@
+---
+metaTitle: "Transforms Quickstart"
+metaDescription: "Quickly get started with transforms using Qri CLI"
+weight: 2
+---
+
+import InfoBlock from '../../../src/components/InfoBlock.js'
+
+This docs page will give you example commands to get started with qri transform scripts using qri CLI.  
+
+## Create a Directory to Work From
+
+Create an empty directory using `mkdir`, `cd` into the new directory.
+
+```bash
+[~/datasets]$ mkdir example-transform
```

Comment:
I find the directory name in the prompt a little confusing. Can we drop them in favor of just `$` to mean "things you should type"?

Reply:
👍 Done.